WebView/app/src/main/assets/app.css

```diff
@@ -43,23 +57,20 @@ img {
   padding: 5px;
 }
 .button {
+  color: black;
```

Comment:
Should the button change in dark theme too?

Reply:
I had it staying black to follow the [Material Dark Theme Guidelines](https://material.io/design/color/dark-theme.html). This guide recommends that the color 'on secondary', which is the color of the button, remains black.